designs/2019-changing-default-of-resolve-plugins-relative-to/README.md

```diff
@@ -0,0 +1,94 @@
+- Start Date: 2019-09-28
+- RFC PR: https://github.com/eslint/rfcs/pull/39
+- Authors: Toru Nagashima ([@mysticatea](https://github.com/mysticatea))
+
+# Changing the Default Value of `--resolve-plugins-relative-to`
+
+## Summary
+
+This RFC changes the default value of `--resolve-plugins-relative-to` CLI option to work on more variety environments.
+
+## Motivation
+
+We can use the config file which is in a common directory by `--config` option. In that case, it's intuitive to load plugins from the directory that contains the config file. But currently, we have to use `--resolve-plugins-relative-to` option along with `--config` option, or have to install required plugins into the current working directory. For example:
+
+```
+eslint . --config /home/me/.eslintrc.js --resolve-plugins-relative-to /home/me
+```
+
+```
+eslint projects/foo --config projects/shared/.eslintrc.js --resolve-plugins-relative-to projects/shared
+```
+
+This behavior is not convenient.
+
+## Detailed Design
+
+### `--resolve-plugins-relative-to` CLI option
+
+This RFC changes the default value of `--resolve-plugins-relative-to` CLI option.
+
+1. If `--resolve-plugins-relative-to` CLI option is present, adopt it.
+1. If `--config` CLI option is present, adopt the directory that contains the config file.
+1. Otherwise, adopt the current working directory.
+
+This setting is used in the whole eslint process. Therefore, every rule ID is identified as unique, so the `rulesMeta` of formatters works fine.
+
+### `resolvePluginsRelativeTo` constructor option of CLIEngine
+
+This RFC changes the default value of `resolvePluginsRelativeTo` option.
+
+1. If `resolvePluginsRelativeTo` constructor option is present, adopt it.
+1. If `configFile` constructor option is present, adopt the directory that contains the config file.
+1. Otherwise, adopt the current working directory.
+
+This setting is used in the whole `CLIEngine` instance. Therefore, every rule ID is identified as unique, so the `CLIEngine#getRules()` method works fine.
+
+### About personal config file (`~/.eslintrc`)
+
+This RFC doesn't change the plugin base path for the personal config file (`~/.eslintrc`) because the functionality has been deprecated in [RFC 32](https://github.com/eslint/rfcs/pull/32). ESLint still loads plugins from the current working directory if there is no project config file, even if there is the personal config file.
+
+If people want to use the personal config file, use `--config` option (e.g., `eslint . --config ~/.eslintrc.json`). It should work intuitively.
+
+### Special handling of `~/`
```

Comment:
Also, just to make it clear (and, yes, I know that I'm coming from nowhere here, and leave comments in a repo I may not belong to).  IMO (and as a Windows user) a) this is YAGNI and probably has more maintenance cost as the core change, b) this is unrelated and should get its own RFC.